### DISCUSSION TITLE
Error building a docker

### DISCUSSION THREAD

**Discussion by Zacc806** [creecros/Mailmagik#35] [Q&A]
Hello all, I've been having trouble installing and making mailmagik work. It did extract the letters from email but never once parsed.
Errors :
PHP Warning: Trying to access array offset on value of type null in /var /www/app/plugins
/Mailmagik/Action/ConvertEmailToTask.php on line 133
confluence-kanboard_petroleum_tst-1
Warning: Trying to access array offset on value of type null in /var/www/app/plugins/Mailmagik/Action/ConvertEmailTo
Task.php on line 133
confluence-kanboard petroleum tst-1 | [04-Aug -2023 11:26:02 UTC] PHP Warning: Trying to access array offset on value of type null in /var /www/app/plugins
/Mailmagik/Action/[ConvertEmatToTask.ph](https://convertemattotask.ph/) on line 13

**Comment by creecros** [comment]
hmmm....
is the user email address that you are sending said email with, a user within the project?

**Comment by Zacc806** [comment]
Hi? sorry for the late reply, i couldnt access the server over the weekends. Where can i configure it?

**Reply by creecros** [reply]
In the project settings panel, under permissions, you can add users to the project. 

In the user setting, for the specific user, you can modify their email.

The user must have permission within said project, and must have the same email from which you are sending the email to create the task.
